test/run-integration-tests

```diff
@@ -60,12 +61,14 @@ prefix() {
 #
 fetch-ringpop-common() {
     if [ ! -e "$ringpop_common_dir" ]; then
-        run git clone --depth=1 https://github.com/uber/ringpop-common.git "$ringpop_common_dir"
+        run git clone --depth=1 https://github.com/uber/ringpop-common.git "$ringpop_common_dir" --branch "$ringpop_common_branch"
```

Comment:
This has been fixed in the integration tests.

With regards to this line, we want to keep it so we can develop to a different branch of the integration tests when we implement new features, but we want to make sure the branch of `ringpop-common` is set to `master` on our `dev` branch (before merging).